pkg/vm/engine/tae/txn/txnbase/impl.go

```diff
@@ -111,24 +114,32 @@ func (txn *Txn) commit2PC() (err error) {
 	state := txn.GetTxnState(false)
 
 	switch state {
+	//It's a 2PC transaction running on Coordinator
 	case txnif.TxnStateCommittingFinished:
 		if err = txn.ApplyCommit(); err != nil {
 			panic(err)
 		}
-		//TODO:Append committed log entry into log service asynchronously
-		//     for checkpointing the committing log entry
-		//txn.SetError(txn.LogTxnEntry())
 		txn.DoneWithErr(nil, false)
+		//Append a committed log entry into log service asynchronously
+		//     for checkpointing the committing log entry
+		_, err = txn.LogTxnState(false)
```

Comment:
1. Free log entry when the log entry is flushed
2. We also need some mechanism to wait for async flush and handle any err